changelog.md

```diff
@@ -67,6 +67,7 @@
 * [2515](https://github.com/zeta-chain/node/pull/2515) - replace chainName by chainID for ChainNonces indexing
 * [2541](https://github.com/zeta-chain/node/pull/2541) - deprecate ChainName field in Chain object
 * [2542](https://github.com/zeta-chain/node/pull/2542) - adjust permissions to be more restrictive
+* [2572](https://github.com/zeta-chain/node/pull/2572) - turn off IBC modules
```

Comment:
maybe we can remove it from the changelog, and not add this 

- https://github.com/zeta-chain/node/pull/2145 - add ibc and ibc-transfer modules


and also change the wording for , to include that its added but not used 
- https://github.com/zeta-chain/node/pull/2154 - add ibccrosschain module

Reply:
Well `ibc-crosschain` has still been implemented but not part of app.go anymore

The work from https://github.com/zeta-chain/node/pull/2145 still here but commented out

Reply:
Yeah, resolving for now, but I feel for someone reading through the changelog it might get confusing to see a module added and then removed in the same release,